project/cmake/scripts/linux/Install.cmake

```diff
@@ -132,6 +132,10 @@ install(FILES ${CORE_SOURCE_DIR}/copying.txt
         DESTINATION ${datarootdir}/doc/${APP_NAME_LC}
         COMPONENT kodi)
 
+install(FILES ${CORE_SOURCE_DIR}/privacy-policy.txt
+        DESTINATION ${datarootdir}${APP_NAME_LC}
+        COMPONENT kodi)
+
```

Comment:
Any reason you didn't used the code block above and install it to ${datarootdir}/doc/${APP_NAME_LC}, same as the other files?